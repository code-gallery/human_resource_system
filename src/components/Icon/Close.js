import React from 'react'
import PropTypes from 'prop-types'

const Close = ({ color, className }) => (
  <svg
    className={className}
    viewBox="0 0 17 17"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g fill="none" fillRule="evenodd">
      <g transform="translate(-958 -160)" stroke={color} strokeWidth="3">
        <g transform="rotate(-45 678.966 -1065.095)">
          <path d="M10,0 L10,20" id="Path-4" />
          <path d="M0,10 L20,10" id="Path-4" />
        </g>
      </g>
    </g>
  </svg>
)

Close.propTypes = {
  color: PropTypes.string.isRequired,
  className: PropTypes.string
}

Close.defaultProps = {
  className: ''
}

export default Close
