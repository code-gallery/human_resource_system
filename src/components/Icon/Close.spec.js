import React from 'react'
import renderer from 'react-test-renderer'
import Close from './Close'

describe('Close Icon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Close color="#000" />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
