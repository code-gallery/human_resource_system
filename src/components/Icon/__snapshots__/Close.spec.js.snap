// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Close Icon renders correctly 1`] = `
<svg
  className=""
  viewBox="0 0 17 17"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <g
      stroke="#000"
      strokeWidth="3"
      transform="translate(-958 -160)"
    >
      <g
        transform="rotate(-45 678.966 -1065.095)"
      >
        <path
          d="M10,0 L10,20"
          id="Path-4"
        />
        <path
          d="M0,10 L20,10"
          id="Path-4"
        />
      </g>
    </g>
  </g>
</svg>
`;
